### DISCUSSION TITLE
mmpretrain --> training accuracy

### DISCUSSION THREAD

**Discussion by thanujan96** [open-mmlab/mmengine#1456] [Q&A]
When i tried to use mmpretrain, since the model outputs only the losses during training loop, i could not find a way to calculate the accuracy of training data set. is there is a specific way to get the accuracy and other metrics for the training loop. 

Also we could not get losses for the validation set.  it would be helpful someone could post here how could i overcome this issue. 

Thanks in advance :)
